portal/apps/org.wso2.carbon.iam.admin-portal.app/src/main/public/js/add.js

```diff
@@ -38,6 +38,17 @@ function displayVals() {
         UUFClient.renderFragment("email-verification", fillingObject,
             "accountVerificationMethod-area", "OVERWRITE", callbacks);
     }
+    else if ($("#verificationSelector option:selected").text() === "With email verification and ask password from user") {
```

Comment:
Is it correct to compare the text for the condition? i think this is very bad practice + don't we need to change these messages according to the locale? In that case how is this going to work ?